src/controllers/moderationactions/ModerationAction.cpp

```diff
@@ -73,6 +73,10 @@ ModerationAction::ModerationAction(const QString &action)
     {
         this->image_ = Image::fromPixmap(getApp()->resources->buttons.ban);
     }
+    else if (action.startsWith("/delete"))
+    {
+        this->image_ = Image::fromPixmap(getApp()->resources->pajaDank);

```

Comment:
I would like to see a proper resource be put here, possibly a trashcan like Twitch uses?

Reply:
Added a trashcan png